### PR TITLE
Fix/multiple orders id duplicate

### PR DESCRIPTION
**Pull request by Nullably**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue:
- Related Clubhouse Story: #1261 

**A description of the changes proposed in the pull request**:
- Create a module for tracking_nonce to ensure its return value is unique, this is necessary due to time.time() resolution on Windows is not enough to create unique order_id 
- Update markets to use this module when creating new order_id
- I have only tested this binance market (as I don't have accounts on other exchanges), the change is minor, I don't expect any issue on other exchanges.

### CODE REVIEW

File: hummingbot/market/bamboo_relay/bamboo_relay_market.pyx [hunk 1]
```diff
@@ -1315,7 +1316,7 @@ cdef class BambooRelayMarket(MarketBase):
                    object price=s_decimal_NaN,
                    dict kwargs={}):
         cdef:
-            int64_t tracking_nonce = <int64_t>(time.time() * 1e6)
+            int64_t tracking_nonce = get_tracking_nonce()
```
Comment (chowryan) [inline hummingbot/market/bamboo_relay/bamboo_relay_market.pyx]: i think`<int64_t>` casts the python `int` to cython `int64_t` so we might want to keep it

File: hummingbot/core/utils/tracking_nonce.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import time
+
+_last_tracking_nonce: int = 0
+
+
+# This tracking nonce is needed bc resolution for time.time() on Windows is very low (16ms),
+# not good enough to create unique order_id.
+def get_tracking_nonce() -> int:
+    global _last_tracking_nonce
+    nonce = (time.time() * 1e6)
```
Comment (chowryan) [inline hummingbot/core/utils/tracking_nonce.py]: this is a float

File: hummingbot/market/binance/binance_market.pxd [hunk 3]
```diff
@@ -1,9 +1,6 @@
-from libc.stdint cimport int64_t
-
 from hummingbot.market.market_base cimport MarketBase
 from hummingbot.core.data_type.transaction_tracker cimport TransactionTracker
 
-
```
Comment (chowryan) [inline hummingbot/market/binance/binance_market.pxd]: our pylint rules check for 2 lines before new class.

**Comment by chowryan** [comment]
minor nitpicks, LGTM

File: hummingbot/core/utils/tracking_nonce.py (hunk 2 above)
Comment (Nullably) [reply hummingbot/core/utils/tracking_nonce.py]: yep, i'll fix this.

File: hummingbot/market/bamboo_relay/bamboo_relay_market.pyx (hunk 1 above)
Comment (Nullably) [reply hummingbot/market/bamboo_relay/bamboo_relay_market.pyx]: i removed the cast as I was worried about performance, after some researching I don't think it's matter much whether to have explicit casting or implicit casting. I'll add it back it.

**Pull request merged**